canonical-web-and-design/maas-ui
# fix(all): add autoprefixer

## Done

- Add autoprefixer to the legacy and root workspaces.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Load the ui in Firefox.
- Open your browser console and make sure that css errors are visible (there are some little toggles near the top).
- You should see some warnings for invalid `-webkit` rules etc. but there shouldn't be any errors for`Error in parsing value for ‘width’.  Declaration dropped.`.

## Fixes

Fixes: #2103.
